igor-web/igor-web.gradle

```diff
@@ -1,51 +1,108 @@
 ext {
   springConfigLocation = System.getProperty('spring.config.location', "${System.getProperty('user.home')}/.spinnaker/")
-  repackage = System.getProperty('springBoot.repackage', "false")
+  versions = [
+    cglib: '3.2.0',
+    kork: '1.119.1',
+    eureka: '1.8.6',
+    groovy: '2.4.13',
+    hamcrest: '1.3',
+    jackson: '2.9.2',
+    junit: '4.12',
+    logstash: '4.11',
+    objenesis: '2.5.1',
+    okHttp: '2.7.0',
+    retrofit: '1.9.0',
+    rxJava: '1.0.16',
+    snakeyaml: '1.15',
+    spock: '1.1-groovy-2.4',
+    spring: '4.3.14.RELEASE',
+    springBoot: '1.5.10.RELEASE',
+  ]
```

Comment:
Wouldn't you want these to be up a level so you could sync across modules?

Reply:
Yeah definitely - there's only one module in Igor   so I was lazy as I was unwinding the spinnaker.dependencies notation